src/samples/nodejs/spi_max31855.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2015 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * This sample code demonstrates the usage of SPI JS bindings for
+ * configuring and reading the temperature in Celsius from the
+ * thermocouple amplifier MAX31855 connected to the MinnowBoard MAX.
+ *
+ * Pin configuration:
+ * The thermocouple amplifier talks over SPI, so connect it to the SPI pins.
+ * Vin -> MinnowBoard MAX pin 4 (3.3V)
+ * GND -> MinnowBoard MAX pin 2 (GND)
+ * DO  -> MinnowBoard MAX pin 7 (MISO)
+ * CS  -> MinnowBoard MAX pin 5 (CSO)
+ * CLK -> MinnowBoard MAX pin 11 (SCLK)
+ *
+ * Press Ctrl+C to exit the process.
+ */
+var spi = require( "soletta/spi" ),
+    spiBus = null;
+
+spi.open( {
+    bus: 0,
+    frequency: 2000000
+} ).then( function( bus ) {
+    spiBus = bus;
+
+    // Take a temperature reading in every one second
+    readInterval = setInterval( function() {
```

Comment:
Shouldn't you have declared this before, like `spi` and `spiBus`? As you use it on signal handler?